bundles/framework/admin-announcements/view/AnnouncementsListHandler.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { StateHandler, controllerMixin, Messaging } from 'oskari-ui/util';
+import { announcementsHelper } from './AnnouncementsHelper';
```

Comment:
This could just be an object named AnnouncementsHelper instead of a function that returns the object.